google-ads-examples/src/main/java/com/google/ads/googleads/examples/campaignmanagement/AddCampaignDraft.java

```diff
@@ -0,0 +1,114 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.ads.googleads.examples.campaignmanagement;
+
+import com.beust.jcommander.Parameter;
+import com.google.ads.googleads.examples.utils.ArgumentNames;
+import com.google.ads.googleads.examples.utils.CodeSampleParams;
+import com.google.ads.googleads.lib.GoogleAdsClient;
+import com.google.ads.googleads.v3.errors.GoogleAdsError;
+import com.google.ads.googleads.v3.errors.GoogleAdsException;
+import com.google.ads.googleads.v3.services.CampaignDraftOperation;
+import com.google.ads.googleads.v3.services.CampaignDraftServiceClient;
+import com.google.ads.googleads.v3.services.MutateCampaignDraftResult;
+import com.google.ads.googleads.v3.services.MutateCampaignDraftsResponse;
+import com.google.ads.googleads.v3.utils.ResourceNames;
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.StringValue;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.UUID;
+
+/**
+ * Adds a campaign draft for a campaign. Make sure you specify a campaign that has a non-shared
```

Comment:
Should the comment about the campaign with a non-shared budget be on the campaign param?

Reply:
This matches the golden

https://github.com/googleads/google-ads-python/pull/251/files

I think it's a reasonable caveat that's worth including here. I've included it below in the javadoc as well.